### ISSUE TITLE
[FR] web interface and docker image

### ISSUE THREAD

**Issue by grasshide** [Tensai75/nzb-monkey-go#22]
I'd love to see this app in a container with a simple web interface!

The main advantages would be:
- dependencies are encapsulated in the container
- available from different clients (doesn't have to be installed on every computer/laptop/phone)
- no dependencies of the used OS (could be used by MAC users as well)
- config could be done in the interface instead of editing files
- go docker images that could serve as the base for this are already available
- does not necessary interfere with the cli version

**Comment by Tensai75** [comment]
Yes, that would certainly be nice, and maybe someone will do it. But I have no intention of doing it at all.

And you should also bear in mind that NZB Monkey Go is supposed to be an automation tool that should start when an NZBLNK is clicked. I don't think this is possible when it is dockerised, at least not without a helper tool that still has to be installed locally to register the NZBLNK URI scheme and send the NZBLNK information to the web interface somehow.

**Comment by nzb-tuxxx** [comment]
I made a proof of concept for a sabnzbd integration allowing to add nzblinks directly from the sabnzbd webui. This wont be merged to upstream but you can patch it yourself, see:

https://github.com/nzb-tuxxx/sabnzbd/tree/feature/nzb_monkey

See attached video preview for details:

![sabnzbd_nzb_monkey_go](https://github.com/Tensai75/nzb-monkey-go/assets/143939649/bf08b2a6-de99-4a9c-937c-ee57d392c397)
